scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -3002,12 +3013,20 @@ def setup(params):
       formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
   # Sets environment variables for MKL
+  # If OS ENV vars are overridden by script defaults, a warning msg is printed.
   if params.mkl:
-    os.environ['KMP_BLOCKTIME'] = str(params.kmp_blocktime)
-    os.environ['KMP_SETTINGS'] = str(params.kmp_settings)
-    os.environ['KMP_AFFINITY'] = params.kmp_affinity
-    if params.num_intra_threads > 0:
-      os.environ['OMP_NUM_THREADS'] = str(params.num_intra_threads)
+    mkl_flags = ['kmp_blocktime', 'kmp_settings', 'kmp_affinity',
+                 'num_intra_threads']
+    for mkl_flag in mkl_flags:
+      os_env_var = mkl_flag.upper()
+      if mkl_flag == 'num_intra_threads':
+        os_env_var = 'OMP_NUM_THREADS'
+      flag_val = str(params.__getattribute__(mkl_flag))
+      if _is_mkl_flag_absent(mkl_flag) and os_env_var in os.environ:
+        _print_os_env_ignored_warning(mkl_flag, flag_val)
+      os.environ[os_env_var] = flag_val
+      if mkl_flag == 'num_intra_threads' and params.num_intra_threads < 1:
```

Comment:
`num_intra_threads` defaults to None, so `params.num_intra_threads < 1` will throw an error. Change this line to
```
if mkl_flag == 'num_intra_threads' and not params.num_intra_threads
```